kk/urls.py

```diff
@@ -11,4 +13,4 @@
     url(r'^v1/', include(router.urls, namespace='v1')),
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api/v1', include('rest_framework.urls', namespace='rest_framework'))
-]
+] + static('/', document_root=settings.BASE_DIR)
```

Comment:
This will leave every single Python file, etc. accessible to the world, won't it?

See [the docs](https://docs.djangoproject.com/en/1.8/howto/static-files/#serving-files-uploaded-by-a-user-during-development) for the canonical way to serve the media root.